linux/nvme.h

```diff
@@ -1458,21 +1458,21 @@ struct nvme_error_log_page {
 struct nvme_firmware_log_page {
 	__u8	afi;
 	__u8	resv[7];
-	__u64	frs[7];
+	__le64	frs[7];
```

Comment:
this type may have incorrectly been carried over from tools implementing pre-releases of the spec. The field is an ASCII array, so it should be a __u8[8].

Reply:
yeah, I have fixed and added a PR https://github.com/linux-nvme/nvme-cli/pull/1088
Please have a look!
Thanks